include/zephyr/bluetooth/buf.h

```diff
@@ -36,6 +36,8 @@ extern "C" {
 
 /** Possible types of buffers passed around the Bluetooth stack in a form of bitmask. */
 enum bt_buf_type {
+	/** Invalid value, used for error checking */
+	BT_BUF_TYPE_INVALID = 0,
```

Comment:
This is a interesting value; technically any value not defined below are invalid, right?

Consider to name this `BT_BUF_TYPE_NONE`